utils/kubernetes/kubeutil.py

```diff
@@ -374,9 +377,17 @@ def get_deployment_for_replicaset(self, rs_name):
         https://github.com/kubernetes/kubernetes/blob/release-1.6/pkg/controller/deployment/sync.go#L299
         But it might change in a future k8s version. The other way to match RS and deployments is
         to parse and cache /apis/extensions/v1beta1/replicasets, mirroring PodServiceMapper
+        In 1.8, the hash generation logic changed: https://github.com/kubernetes/kubernetes/pull/51538/files
```

Comment:
i'd like to see a note here that we may have a few false positive. It's okay that we do until we have the cluster agent, but we should document it somewhere and what better place than where it happens?

Reply:
Added comment